Buildings/Examples/VAVReheat/BaseClasses/VAVBranch.mo

```diff
@@ -11,8 +11,20 @@ model VAVBranch "Supply branch of a VAV system"
 
   parameter Modelica.SIunits.MassFlowRate m_flow_nominal
     "Mass flow rate of this thermal zone";
+  parameter Real ratVFloHea = 1.0 "Flow rate ratio in heating mode";
   parameter Modelica.SIunits.Volume VRoo "Room volume";
 
+  parameter Modelica.SIunits.Temperature T_HW_nominal=355.35 "Reheat coil nominal inlet water temperature";
+
+  final parameter Modelica.SIunits.Temperature TAirInlNom = 281.65 "Inlet air nominal temperature";
+  final parameter Modelica.SIunits.Temperature TAirOutNom = 323.15 "Outlet air nominal temperature";
+  final parameter Modelica.SIunits.SpecificHeatCapacity CpAir = 1004 "Air specific heat capacity";
+  final parameter Modelica.SIunits.SpecificHeatCapacity CpWater = 4180 "Water specific heat capacity";
+  final parameter Modelica.SIunits.HeatFlowRate QFloNom = m_flow_nominal*ratVFloHea*CpAir*(TAirOutNom - TAirInlNom) "Nominal heat flow rate";
+  final parameter Modelica.SIunits.TemperatureDifference dTHWNom = 10 "Nominal hot water temperature delta";
+
+  final parameter Modelica.SIunits.MassFlowRate m_flow_HW_nominal = QFloNom/(CpWater * dTHWNom)  "Mass flow rate of hot water to reheat coil";
```

Comment:
Please use `*_flow_nominal` for heat/mass flow rates at design (nominal) condition.
Also, use lower case for `cpWater` (or 3 letter camel case as in `cpWat` where `c` is lower case because it is a variable name, and we use `cp` because that is a commonly used variable in the literature and hence commonly understood as opposed to `speHeaCap`)